sdk/storage/Azure.Storage.DataMovement.Blobs/src/AppendBlobStorageResource.cs

```diff
@@ -27,17 +27,17 @@ public class AppendBlobStorageResource : StorageResource
         private long? _length;
 
         /// <summary>
-        /// Returns URL
+        /// Gets the URL of the storage resource.
         /// </summary>
         public override Uri Uri => _blobClient.Uri;
 
         /// <summary>
-        /// Gets the path of the resource.
+        /// Gets the path of the storage resource.
         /// </summary>
         public override string Path => _blobClient.Name;
 
         /// <summary>
-        /// Defines whether the object can produce a SAS URL
+        /// Defines whether the storage resource type can produce a SAS URL.
```

Comment:
Sorry, wanted to call out this one to make sure this is accurate (i.e. does tell you about SAS URL viability)

Reply:
Nah. I think this was a mistake on my end. It used to be a SAS url, but here we just return only the URL. So we can omit SAS from the comment.

Reply:
Sounds good. Fixed all occurrences from `SAS URL` to just `URL` in this lib